README.md

```diff
@@ -1,5 +1,6 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_tree-from-object-DOM/)
+    - [DEMO LINK](https://Vitaliy-Anochin
+.github.io/js_tree-from-object-DOM/)
```

Comment:
```suggestion
    - [DEMO LINK](https://Vitaliy-Anochin.github.io/js_tree-from-object-DOM/)
```

Reply:
![image](https://user-images.githubusercontent.com/38065883/174626013-6d624b71-5330-4e10-a2fe-80fbfa931d6d.png)

Same here

https://user-images.githubusercontent.com/38065883/144253618-cab69f76-2bad-470e-a1e4-153a08221108.mov

Reply:
<img width="1073" alt="image" src="https://user-images.githubusercontent.com/38065883/175118692-f6e4bf8e-9e6f-4e03-94bd-584a9b73baf2.png">
There is still no working demo page